.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+
+cache: yarn
+
+sudo: false
+
+node_js:
+  - '7'
+
+after_success:
+  - npm run deploy
```

Comment:
can you change this line to:
```
  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run deploy; fi'
```

Reply:
Like this?

    node_js:
      - '7'

    after_success:
      - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run deploy; fi'